### ISSUE TITLE
Question: Login into SAP without captcha and cookies?

### ISSUE THREAD

**Issue by BenjaminVeit** [ui5-community/wdi5#481]
Hay,

I am currently working on automated tests for a SAP Fiori application. I am using the WDI5 framework for this. Unfortunately, as soon as I call my application via the test, I have the problem that I have to log in to SAP and accept cookies. Logging in itself was not a problem at the beginning, but as soon as I have logged in for example 5x, I have to solve a captcha.

Do you happen to know how I can solve the problem with the captcha and cookies or is there a solution in the WDI5 framework?
I have already researched a lot but have not found a solution yet.

Thanks in advance!

**Comment by vobu** [comment]
yeah, we hear this quite often that with the default SAP Cloud IdP on BTP, the Captcha breaks automation flows.
Unfortunately, there is no reliable remedy in `wdi5` for that as of now 😢 
But: thanks to the sublime debug powers of @jasper07, we were able to [introduce a way to circumvent the biometric auth prompt in Work Zone](https://github.com/ui5-community/wdi5/pull/447/files#diff-b0669afc14454e1a320fe0b031d3f00d12b88d399ff809cf1a5ef191278805e5R23).
Now, I wonder: when the Captcha prompt appears - and you look at the cookies 🍪 in the debug console - might there be a similar identifiable cookie that could be used in `wdi5` to circumvent the Captcha prompt?
If you find sth here, I'd be game to help pour this into code...

**Comment by BenjaminVeit** [comment]
The problem with cookies is that they do not always appear. The last time I tried, they were no longer there. I don't know what the reason is.

Unfortunately I did not find anything about the cookies regarding the captcha. 

However, I have another question, is it possible to use both the BTP authentication variant and the Basic authentication at the same time? That is I log in directly with BTP at the beginning and then log in with Basic Authentication to another tool on the website
